Lib/collections/__init__.py

```diff
@@ -85,9 +85,8 @@ class OrderedDict(dict):
 
     def __init__(*args, **kwds):
         '''Initialize an ordered dictionary.  The signature is the same as
-        regular dictionaries, but keyword arguments are not recommended because
-        their insertion order is arbitrary.
-
+        regular dictionaries.  As of Python 3.6, keyword argument insertion
```

Comment:
Should I mention that keyword order is preserved? Or just drop that altogether and fall back to the compatibility claim alone?